pdfPreviewInMsg/initial.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * This example extension is based on work from:
+ *  - e-gaulue
+ *  - rholeczy
+ */
+
+
+// Listener for messages received from the background.
+browser.runtime.onMessage.addListener((data, sender) => {
+  if (data.command == "hideSpinner") {
+    document.getElementById("loader").setAttribute("style", "display:none");
+  }
+
+  if (data.command == "prepareMessageBody") {
+    let originalBody = document.body.innerHTML;
+    document.body.innerHTML = "";
+
+    let mainDiv = document.createElement("div");
+    mainDiv.setAttribute("style", "display: flex;");
+    document.body.appendChild(mainDiv);
+    mainDiv.id = "mainDiv";
+
+    let firstDiv = document.createElement("div");
+    mainDiv.appendChild(firstDiv);
+    firstDiv.setAttribute("style", "flex-grow: 1");
+    firstDiv.innerHTML = originalBody;
```

Comment:
We need to find an alternative solution for this.

Reply:
We divide the mail content in "two tabs" the first contains the orignal body, the second contains our images and pdf preview. For the firstDiv contains the original body you don't want to use innerHTML ?

Reply:
In this specific case it is not an unsafe assignment to innerhtml, as you just copy the existing body, but since this goes on the official sample repo, I would like to find a different solution. Either by just appending a float div to the end of the body or using https://developer.mozilla.org/de/docs/Web/API/Element/insertAdjacentHTML

I will play with that tomorrow.

Reply:
Ok no problem, if i find something i will contact you here, but i'm in internship so i have work beside

Reply:
So if i don't give answer, sorry

Reply:
You have already done great, this is on me. Just wanted to inform you, why it has not been merged yet.